addons/website_sale_slides/views/website_sale_templates.xml

```diff
@@ -30,7 +30,8 @@
                            <t t-if="course.total_time"> - </t><t t-out="course.total_slides"/> step(s)
                         </t>
                     </div>
-                    <a role="button" class="btn btn-primary ms-2 my-2" t-attf-href="/slides/#{slug(course)}">
+                    <t t-set="signup_token" t-value="order.partner_id.signup_prepare() and order.partner_id.signup_token"/>
+                    <a role="button" class="btn btn-primary ms-2 my-2" t-attf-href="/web/login?token=#{signup_token}&amp;redirect=/slides/#{slug(course)}">
```

Comment:
Seems a bit overly simplistic but it passed testing so I'm guessing it's working as intended :thinking: 

I know that starting from 16.3 the flows have been reworked by #70291
Can you check if it's working there? If not, maybe we should use "slide_channel_partner.invitation_link" if possible as more integrated in the flows? Unless that relationship record is not available in this template?

We can rework it during the FW if needed, no need to make a separate PR.

Reply:
Hello @std-odoo 

I meant, is it working without your commit, but I'm guessing it's not.

Then I'm thinking it's better starting from 16.3 to use create and use an "invitation_link", as it contains all the logic we need to prepare the signup and redirect accordingly, correct?
But we can keep this "simple fix" until that version.

Hereby summoning @Odoonan to make sure I'm not completely wrong 😄 

(Again, don't need to make a separate PR, we can rework that during FW, I would just like to confirm it now).

Reply:
ah ok sorry :) Yes the bug exists in 16.3 without my commit

> to use create and use an "invitation_link"

good idea :)

Reply:
Small ui detail as I am thinking about it:

If I am not mistaken, /login page will land on the login page and will only have "reset password" if the option Free Sign Up is not activated in the settings. If it is activated, "Create an account" will also show. Using the /invite route will handle all that and redirect directly to the signup page if needed.

Have a nice afternoon :)